src/server/views/app.handlebars

```diff
@@ -111,6 +109,26 @@
         .catch(console.error);
     }
 
+    function updateFetchDocumentUrlElement(value) {
+      document.getElementById('fetch-document-url').value = value;
+    }
+
+    const urlParams = new URLSearchParams(window.location.search);
+
+    if (urlParams.get('load')) {
+      const asyncApiUrl = urlParams.get('load')
+
+      updateFetchDocumentUrlElement(asyncApiUrl)
+      loadAsyncApi(asyncApiUrl)
+    }
+
+    if (urlParams.get('url')) {
+      const asyncApiUrl = urlParams.get('url')
+
+      updateFetchDocumentUrlElement(asyncApiUrl)
+      loadAsyncApi(asyncApiUrl)
+    }
+
```

Comment:
I'm always a fan of the approach to have as less code as possible, of course as less as it is still easy to read. What do you think about 👇  code?

```js
    const urlParams = new URLSearchParams(window.location.search);
    const asyncApiUrl = urlParams.get('load') || urlParams.get('url');
    
    if (asyncApiUrl) {
      document.getElementById('fetch-document-url').value = asyncApiUrl;
      loadAsyncApi(asyncApiUrl);
    }
```

Reply:
You're right it's more succinct and still very readable 👍